cvise_reduce.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -eu
+
+# UPDATE: YKLUA path
+YKLUA="/home/pd/yklua-fork-cvise"
```

Comment:
I don't think this path will work for the rest of us ;)

Reply:
That's why I added instructions in README :)
But I couldn't think of a better solution. 
Alternatively, we can set some kind of global config to run `cvise_reduce.sh`. Environment variables set when running `cvise` don't seem to be available in 'cvise_reduce.sh`.